flutter/beagle_components/lib/beagle_text.dart

```diff
@@ -51,12 +53,21 @@ class BeagleText extends StatelessWidget {
     } else if (alignment == TextAlignment.LEFT) {
       return TextAlign.left;
     } else {
-      return defaultTextAlign;
+      return null;
     }
   }
 
   Color getTextColor(String color) {
-    return HexColor(color ?? defaultTextColor);
+    return color != null ? HexColor(color) : null;
+  }
+
+  TextStyle getTextStyle() {
+    var textStyle = _designSystem?.textStyle(styleId);
+    textStyle ??= const TextStyle();
```

Comment:
```suggestion
    var textStyle = _designSystem?.textStyle(styleId) ?? const TextStyle();
```
This is not a better way, it is just a suggestion to do the same thing. You decide.